tools/README.md

```diff
@@ -4,11 +4,11 @@ If you want to deploy the CNF-Testbed in the same way that our CI job does, you
 can do that by using the Makefile and following the steps below.
 
 ## Pre-requisites:
-Set your Packet.net project name, id & api tokens.
+Set your [Equinix Metal] project name, id & api tokens. These can be found through the [Console].
```

Comment:
Did we want to add a URL for the [Console]?

Reply:
Please disregard, I see below: 
- [Console]: http://console.equinix.com/ "Console"

LGTM